onnx/defs/tensor/defs.cc

```diff
@@ -2088,7 +2093,7 @@ ONNX_OPERATOR_SET_SCHEMA(
             static_cast<int64_t>(1))
         .TypeConstraint(
             "T1",
-            {"tensor(float)", "tensor(double)"},
+            {"tensor(float)", "tensor(double)", "tensor(bfloat16)"},
```

Comment:
I don't think we need bfloat16 for this operator. At least, it doesn't support float16 now.

Reply:
removed